### PR TITLE
Launch IPFS daemon

### PR DESCRIPTION
**Pull request by emilevauge**
This PR allows to launch IPFS daemon from driver directly ;)

### CODE REVIEW

File: driver.go [hunk 1]
```diff
@@ -36,7 +35,7 @@ func (d ipfsDriver) Create(r volume.Request) volume.Response {
 		return volume.Response{}
 	}
 
-	volumePath := filepath.Join(d.mountPoint, volumeName)
+	volumePath := volumeName
```
Comment (vdemeester) [inline driver.go]: Why this change ?

Comment (emilevauge) [inline driver.go]: Because you can then have the possibility to use [IPNS](https://github.com/ipfs/examples/tree/master/examples/ipns): `docker run -it --rm -v /ipns/QmdiHSUmmhMj9tNx5tKqsoJcen2jeUHjKwGeR8EniaZJ4u:/data --volume-driver=ipfs busybox cat data`. For exemple: https://gateway.ipfs.io/ipns/QmdiHSUmmhMj9tNx5tKqsoJcen2jeUHjKwGeR8EniaZJ4u

Comment (vdemeester) [inline driver.go]: oohh but `-v /ipns/Qm…:/data` is just a bind mount though.
But I see where you are going though :stuck_out_tongue_closed_eyes:

Comment (emilevauge) [inline driver.go]: ^^ True that...
Maybe it would be better to add a second plugin:
`docker run -it --rm -v QmdiHSUmmhMj9tNx5tKqsoJcen2jeUHjKwGeR8EniaZJ4u:/data --volume-driver=ipns busybox cat data`

Comment (vdemeester) [inline driver.go]: or `docker run -it --rm -v ipns:Qmdi… --volume-driver=ipfs …`, `docker run -it --rm -v ipfs:Q…`
